### PR TITLE
Always have dynamic-parsers config file

### PR DESCRIPTION
**Pull request by rakesh-garimella**
**Description**

Changes proposed in this pull request:

- Have empty dynamic parsers file always present
- Remove reference to parse.conf (in path /etc/parser.conf) as we dont use it

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/16733

### CODE REVIEW

File: components/telemetry-operator/internal/resources/logpipeline/resources.go [hunk 1]
```diff
@@ -433,6 +432,17 @@ func MakeConfigMap(name types.NamespacedName) *corev1.ConfigMap {
 	}
 }
 
+func MakeDynamicParserConfigmap(name types.NamespacedName) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-luascripts", name.Name),
```
Comment (chrkl) [inline components/telemetry-operator/internal/resources/logpipeline/resources.go:438]: Guess the name is a copy/paste error.

File: components/telemetry-operator/controller/logpipeline/controller.go [hunk 2]
```diff
@@ -188,6 +188,10 @@ func (r *Reconciler) reconcileFluentBit(ctx context.Context, name types.Namespac
 	if err := utils.CreateOrUpdateConfigMap(ctx, r, luaCm); err != nil {
 		return fmt.Errorf("failed to reconcile fluent bit lua configmap: %w", err)
 	}
+	parsersCm := resources.MakeDynamicParserConfigmap(name)
+	if err := utils.CreateIfNotExistsConfigMap(ctx, r, parsersCm); err != nil {
```
Comment (chrkl) [inline components/telemetry-operator/controller/logpipeline/controller.go:192]: Let's validate the creation of the CM in the controller_test.

**Review by chrkl — APPROVED** [review]

**Review by skhalash — CHANGES_REQUESTED** [review]
IMHO, this functionality belongs to the LogParser controller

**Review by skhalash — APPROVED** [review]

**Pull request merged**